README.md

```diff
@@ -319,6 +319,97 @@ You can use dbt-coves to improve the standard dbt docs generation process. It ge
 # Mandatory when using --merge-deferred
 ```
 
+### Generate airflow-dags
+
+```console
+dbt-coves generate airflow-dags
+```
+
+Translate YML files into their Airflow Python code equivalent. With this, DAGs can be easily written with some `key:value` pairs.
+
+The basic structure of these YMLs must consist of:
+
+- Global configurations (description, schedule_interval, tags, catchup, etc.)
+- `default_args`
+- `nodes`: where tasks and task groups are defined
+  - each Node is a nested object, with it's `name` as key and it's configuration as values.
+    - this configuration must cover:
+      - `type`: 'task' or 'task_group'
+      - `operator`: Airflow operator that will run the tasks (full _module.class_ naming)
+      - `dependencies`: whether the task is dependent on another one(s)
+      - any `key:value` pair of [Operator arguments](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/operators/index.html)
+
+Basic YML DAG example:
+
+```yaml
+description: "dbt-coves DAG"
+schedule_interval: "@hourly"
+tags:
+  - version_01
+default_args:
+  start_date: 2023-01-01
+catchup: false
+nodes:
+  task_1:
+    type: task
```

Comment:
I'd remove the `type` key as it's optional.

Reply:
What if the `basic yml dag` is at the end of all the generate-airflow-dags docs, and more complete? Adding some task_group with generator and tasks-list, an example of `AirbyteDbtGenerator`, etc.

It will be a long yml, but I believe it all makes sense once you see how the YMLDag is built

Reply:
I'd try to make the README simple, with links to a more complete documentation, you might need to create extra files under /docs, as we did in datacoves.